### PR TITLE
PN-355: Matching notifications page: By default have rejected option unchecked; PN-354: Matching notification page: modify default settings for "Matches for patients owned by"

### PR DESCRIPTION
**Pull request by veronikaslc**
Done.

### CODE REVIEW

File: matching-notification-ui/src/main/resources/MatchingNotification/MatchesTable.xml [hunk 1]
```diff
@@ -138,14 +138,14 @@ $xwiki.jsx.use('PhenoTips.Widgets')##
...
               &lt;/li&gt;
               &lt;li&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.ownershipFilter'))&lt;br&gt;
                 &lt;label&gt;&lt;input name="ownership-filter" type="checkbox" value="me" checked&gt; $escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.owned_by_me'))&lt;/label&gt;&lt;br&gt;
                 &lt;label&gt;&lt;input name="ownership-filter" type="checkbox" value="group" checked&gt; $escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.owned_by_group'))&lt;/label&gt;&lt;br&gt;
-                &lt;label&gt;&lt;input name="ownership-filter" type="checkbox" value="others" #if(!$hasAdmin)checked#end&gt; $escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.owned_by_other'))&lt;/label&gt;&lt;br&gt;
+                &lt;label&gt;&lt;input name="ownership-filter" type="checkbox" value="others" checked&gt; $escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.owned_by_other'))&lt;/label&gt;&lt;br&gt;
```
Comment (sdumitriu) [inline matching-notification-ui/src/main/resources/MatchingNotification/MatchesTable.xml]: Isn't the intended behavior to have this checked **only** for admins? That means simply removing the `!` from the condition, not the whole condition.

**Comment by sashaandjic** [comment]
I would keep Other option turned on by default for everyone. When Brittney logged this task, her main concern were numerous matches where on one side had a public patient. With Public option turned off by default, they will go away. I believe having ability to see Other by default is beneficial as you will see matches for all the patient cases you have access to. Users can always turn off this option if they want to see only cases owned by them.

**Comment by sashaandjic** [comment]
After talking to Brittney, we realized Other should be turned off by default for typical users.
Global admins and group admins should have it turned on by default.

**Comment by veronikaslc** [comment]
Addressed codereview comments.

**Comment by veronikaslc** [comment]
Re-tested & re-approved by Sasha.

**Pull request merged**